pkg/ssf_events/ssf_event.go

```diff
@@ -137,6 +143,48 @@ func EventStructFromEvent(eventUri string, eventSubject interface{}, claimsJson
 			CurrentStatus:  currentStatus,
 		}
 		return &event, nil
+
+	case AssuranceLevelChange:
+		previousLevel, ok := subjectAttributes["previousLevel"].(string)
+		if !ok {
+			return nil, errors.New("unable to parse previous level")
```

Comment:
We should allow creation of "assurance level change" events without the previous level populated.